visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.tsx

```diff
@@ -429,8 +519,10 @@ export class CohortEditor extends React.PureComponent<ICohortEditorProps, ICohor
                 )}
                 {!selectedMeta.treatAsCategorical && (
                     <div className={styles.featureTextDiv}>
-                        <Text variant={"small"} className={styles.featureText}>
-                            {`Min: ${selectedMeta.featureRange.min}`} {`Max: ${selectedMeta.featureRange.max}`}
+                        <Text block nowrap variant={"small"} className={styles.featureText}>
+                            {localization.Filters.min}{(selectedMeta.featureRange.min % 1) !=0 ? (Math.round(selectedMeta.featureRange.min * 10000) / 10000).toFixed(4): selectedMeta.featureRange.min}
+                            {localization.Filters.space}
```

Comment:
same as above, template literal remove the need for localization.space and can move the formatting of numbers to be before. Can any of the code be reused between this one and the other min/max label?

Reply:
computing the min and max value in roundDecimalValue()